test/DynamoCoreUITests/DynamoTestUIBase.cs

```diff
@@ -17,6 +17,8 @@ namespace DynamoCoreUITests
 {
     public class DynamoTestUIBase
     {
+        private Preloader preloader;
+
```

Comment:
Unit tests now rely on `DynamoShapeManager.Preloader` for its geometry preloading activities.